src/new-components/tabs/tabs.tsx

```diff
@@ -125,6 +125,12 @@ type TabListProps = (
            */
           'aria-labelledby': string
       }
+    | {
+          /**
+           * For cases where multiple instances of the tab list exists, the duplicates may be marked as aria-hidden
```

Comment:
What do these cases (where multiple instances of the tab list exists) look like?

Reply:
I have a use case for [rendering the tab buttons on the thread list's sticky header](https://paper.dropbox.com/doc/Inbox-1.1-Done-tabview--BQ2nJwTOQGKh2v0k3L7_RpU9Ag-Lwc0Vs9AnWb3BA6uLy3VX#:h2=Scrolling)

What I'm thinking is that this `<TabList>` will sit inside of a `<TabPanel>`. I tested with a simple case in storybook and it seems to re-render correctly 🤞